tarantool/connection.py

```diff
@@ -1751,3 +1765,765 @@ def _check_features(self):
         features_list = [val for val in CONNECTOR_FEATURES if val in server_features]
         for val in features_list:
             self._features[val] = True
+
+    def crud_insert(self, space_name: str, values: Union[tuple, list], opts={}) -> CrudResult:
+        """
+        Inserts row through the `tarantool-crud <https://github.com/tarantool/crud#insert>`__ module.
```

Comment:
I'm sorry that we don't have a linter for now.

PEP8 [declares](https://peps.python.org/pep-0008/#maximum-line-length) that
> For flowing long blocks of text with fewer structural restrictions (docstrings or comments), the line length should be limited to 72 characters.

This restriction **includes** the tabulation before docstings. There are many places in this package where this rule is violated, but in all of them we simply can't do it any other way.

For example, you can solve the issue here by moving the link in a separate line, like this
https://github.com/tarantool/tarantool-python/blob/70b0a3debe5ffa7f1c0b5c84d6fb9e28cbe26152/tarantool/connection.py#L469

Reply:
Done